src/wp-includes/class-wp-block-styles-registry.php

```diff
@@ -60,6 +60,15 @@ public function register( $block_name, $style_properties ) {
 			return false;
 		}
 
+		if ( strpos( $style_properties['name'], ' ' ) !== false ) {
```

Comment:
This could be `str_contains` instead to make it more readable:

```suggestion
		if ( ! str_contains( $style_properties['name'], ' ' ) ) {
```

Core now has a polyfill for using `str_contains()` https://www.php.net/manual/en/function.str-contains.php.